docs/release-notes/8.0.asciidoc

```diff
@@ -1,6 +1,55 @@
 [[release-notes-header-8.0.0]]
 == 8.0
 
+[discrete]
+[[release-notes-8.0.0]]
+=== 8.0.0
+
+[discrete]
+[[post-upgrade-reqs-8.0.0]]
+==== Post upgrade requirements
```

Comment:
Not sure if there's a team/Elastic guideline, but in general I'd hyphenate this, since it's acting as an adjective.
```suggestion
==== Post-upgrade requirements
```

Reply:
Yeah, I was following the way it was written in the [7.10 RN](https://www.elastic.co/guide/en/security/7.13/release-notes.html#release-notes-7.10.0), but then realized we hyphenate it in our main docset (see https://www.elastic.co/guide/en/security/master/post-upgrade-intro.html), so I agree with this edit.